tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "strict": true,
     "esModuleInterop": true,
     "outDir": "dist",
-    "declaration": true
+    "declaration": true,
+    "resolveJsonModule": true
```

Comment:
Was this needed to load the ABI file?

Reply:
Yeah, Since all the ABI files were removed in #18. This can also be removed during cleanup.